### ISSUE TITLE
Veterans Crisis Line page accessibility issues

### ISSUE THREAD

**Issue by esther-k** [department-of-veterans-affairs/va.gov-team#16238]
## What Happened?
1. With Keyboard linked to Android: the "VeteransCrisisLine.net" hyperlink is not being focused and can not be accessed 
2. With Voice Access enabled on Android, the "VeteransCrisisLine.net" hyperlink is not able to be accessed by saying the attached number (the hyperlink has no attached number) or by saying "touch Veterans Crisis Line.net"
3. With VoiceOver and TalkBack enabled on iOS and Android, the back button a11yHint is not being read
     - NOTE: Should be updated with [a11y update ticket](https://app.zenhub.com/workspaces/va-mobile-app-5f205eac3155a2000f422ecb/issues/department-of-veterans-affairs/va.gov-team/17185)

## Specs:
- Device: Android - LG LM-Q720 Stylo 5 with Voice Access and TalkBack enabled
- Device: iOS - iPhone XR with VoiceOver enabled
- Bluetooth Keyboard

## Steps to Reproduce

For 1:
1. With Keyboard linked to Android, navigate to the Veterans Crisis Line page 
2. Press tab to get the keyboard focus on the back button
3. Press tab or the down arrow key. The Call 800-273-8255 and press 1 hyperlink will be in focus
4. Press tab or the down arrow key. The Text 838255 hyperlink will be in focus
5. Press tab or the down arrow key. The Start a confidential chat hyperlink will be in focus
6. Press tab or the down arrow key. The 800-799-4889 hyperlink will be in focus
7. Press tab or the down arrow key. The Home nav tab will be in focus (skipping the VeteransCrisisLine.net hyperlink)

For 2: 
1. With Voice Access enabled on Android, navigate to the Veterans Crisis Line page 
2. Press the blue Voice Access button to enable Voice Access on the page
3. Note that the "VeteransCrisisLine.net" hyperlink is not able to be accessed by saying an attached number (the hyperlink has no attached number) or by saying "touch VeteransCrisisLine.net"

## Desired Behavior
For 1:
- With Keyboard linked to Android, the "VeteransCrisisLine.net" hyperlink will be able to be placed in focus and selected with the tab or navigational keys

For 2:
- Using Voice Access on Android, the "VeteransCrisisLine.net" hyperlink will be able to be selected by saying the attached number or by saying "touch VeteransCrisisLine.net"

## Acceptance Criteria
See desired behavior

## Bug Severity
- 4 - Low

## Linked Story
- Story #14016

## Screen Shot (s) / Recordings (s) and Additional Information
1. [Video of the keyboard bug on Android](https://drive.google.com/file/d/1-yiFVWNIaZRPAZXvhRfrdZBpUz5Y2ORa/view?usp=sharing)
2. Note that there is no attached number to the hyperlink  🔽
![Veterans Crisis Line Hyperlink.png](https://images.zenhubusercontent.com/5f7bb216620a259ab7e50d60/259c89de-dab6-48b8-b120-cd154d339b09)

**Comment by digitalken** [comment]
PASS 1/8/2021 - All items on screen are accessible with voice access.
